virtc/main.go

```diff
@@ -100,6 +100,18 @@ var podConfigFlags = []cli.Flag{
 		Value: "",
 		Usage: "the hyperstart tty socket type",
 	},
+
+	cli.StringSliceFlag{
+		Name:  "hyper-mount-tag-list",
+		Value: new(cli.StringSlice),
+		Usage: "the hyperstart mount tag list",
+	},
+
+	cli.StringSliceFlag{
+		Name:  "hyper-host-path-list",
+		Value: new(cli.StringSlice),
+		Usage: "the hyperstart host path list",
+	},
```

Comment:
I have updated the commit. It now expects an entry like this: -volume="mounttag1:hostpath1 mounttag2:hostpath2 mounttag3:hostpath3"